src-11ty/index.html

```diff
@@ -18,6 +17,11 @@
     font-size: 12px;
     line-height: 1em;
   }
+  .instance {
+    display: flex;
+    padding: 8px;
```

Comment:
To fix 
![image](https://user-images.githubusercontent.com/1444314/195938617-177640db-4d2f-4c92-9f1c-5779fdfc2cf6.png)

```suggestion
    padding: 8px 0;
```